python_src/src/lamp_py/performance_manager/l1_cte_statements.py

```diff
@@ -116,12 +125,12 @@ def get_rt_trips_cte(start_date: int) -> sa.sql.selectable.CTE:
 
     return (
         sa.select(
-            VehicleEvents.fk_static_timestamp,
-            VehicleEvents.direction_id,
-            VehicleEvents.route_id,
-            VehicleEvents.start_date,
-            VehicleEvents.start_time,
-            VehicleEvents.vehicle_id,
+            VehicleTrips.fk_static_timestamp,
+            VehicleTrips.direction_id,
+            VehicleTrips.route_id,
+            VehicleTrips.start_date,
+            VehicleTrips.start_time,
+            VehicleTrips.vehicle_id,
```

Comment:
Pull these values from `VehicleTrips` so that they columns can be dropped from the `VehicleEvents` table.